packages/api/src/controllers/user.test.ts

```diff
@@ -404,29 +409,26 @@ describe("controllers/user", () => {
       // adding emailValid true to user
       await db.user.update(nonAdminUser.id, { emailValid: true });
 
-      // should return admin priviledges error
       client.apiKey = nonAdminApiKey;
       res = await client.get("/user");
       resJson = await res.json();
-      expect(res.status).toBe(403);
-      expect(resJson.errors[0]).toBe("user does not have admin priviledges");
+      condition = Array.isArray(resJson);
+      expect(res.status).toBe(200);
+      expect(condition).toBe(false);
 
       // adding emailValid true to admin user
       await db.user.update(adminUser.id, { emailValid: true });
 
-      // only jwt auth should be allowed access to this list API
       client.apiKey = adminApiKey;
       res = await client.get("/user");
       resJson = await res.json();
-      expect(res.status).toBe(403);
-      expect(resJson.errors[0]).toBe("user does not have admin priviledges");
+      expect(res.status).toBe(200);
 
       client.apiKey = undefined;
       client.basicAuth = `${adminUser.id}:${adminApiKey}`;
       res = await client.get("/user");
       resJson = await res.json();
-      expect(res.status).toBe(403);
-      expect(resJson.errors[0]).toBe("user does not have admin priviledges");
+      expect(res.status).toBe(200);
```

Comment:
Should probably check if the response is the own user JSON. Maybe check only the ID?